template-compose/detekt-config.yml

```diff
@@ -189,6 +189,7 @@ naming:
   FunctionNaming:
     active: true
     functionPattern: '^([a-z$][a-zA-Z$0-9]*)|(`.*`)$'
+    ignoreAnnotated: [ 'Composable' ]
```

Comment:
I don't see this attr in sample project config 🤔

Reply:
@Wadeewee please adding missing config then I can approve this PR 💁‍♂️

Reply:
@luongvo It has been added already in some previous PR checks at line `192` in `sample-compose` 

![Screen Shot 2565-12-16 at 09 06 33](https://user-images.githubusercontent.com/28002315/208005811-b041fcdf-e5d7-45cb-8506-4d52c608680a.png)